query-algebrizer/src/errors.rs

```diff
@@ -42,6 +48,21 @@ error_chain! {
             display("unbound variable: {}", name)
         }
 
+        InvalidBinding(function: PlainSymbol, binding_error: BindingError) {
+            description("invalid binding")
+            display("invalid binding for {}: {:?}.", function, binding_error)
+        }
+
+        GroundBindingsMismatch {
+            description("mismatched bindings in ground")
+            display("mismatched bindings in ground")
+        }
+
+        InvalidGroundConstant {
+            // TODO: flesh this out.
```

Comment:
File follow-up to enrich all of these error types.  I have the same process to come in the transactor, boo :(